tests/test_survey_schema.py

```diff
@@ -15,176 +15,183 @@ def test_get_messages(self):
 
     def test_get_titles_messages(self):
         schema = SurveySchema({
-            "type": "CalculatedSummary",
-            "id": "block-3",
-            "titles": [{
-                "value": "Calculated Summary Main Title Block 2"
-            }],
-            "calculation": {
-                "calculation_type": "sum",
-                "answers_to_calculate": [
-                    "first-number-answer",
-                    "second-number-answer"
-                ],
-                "titles": [{
-                    "value": "Calculated Summary Calculation Title Block 2"
-                }]
-            }
+            "sections": [{
```

Comment:
I wonder if there should be explicit unit tests that include variants or piped content as part of this change?